# HG changeset patch
# User Christoph Junghans <junghans@votca.org>
# Date 1407773183 21600
#      Mon Aug 11 10:06:23 2014 -0600
# Node ID ee313041ba5d7f0706f8140b4e965179e807ef56
# Parent  c66c1be9d40ae69aa61c45572fbe14ef3209378e
Fix usage of LDSHARED

- LDSHARED contain the linker, too
- was used as it would only contain the options in the latter part
- use split_linker_cmd to throw away the linker cmd
- see https://bugs.gentoo.org/show_bug.cgi?id=514288

diff -r c66c1be9d40a -r ee313041ba5d conf/mpidistutils.py
--- a/conf/mpidistutils.py	Wed Sep 04 22:40:25 2013 +0300
+++ b/conf/mpidistutils.py	Mon Aug 11 10:06:23 2014 -0600
@@ -176,6 +176,7 @@
         opt        = environ.get('OPT',        opt       )
         ccshared   = environ.get('CCSHARED', ccshared)
         ldshared   = environ.get('LDSHARED', ldshared)
+        _, ldshared = split_linker_cmd(ldshared)
         cflags     = ' '.join((basecflags, opt, cflags))
         cxxflags   = ' '.join((basecflags, opt, cxxflags))
         cxxflags = cxxflags.replace('-Wstrict-prototypes', '')
